src/main/java/com/couchbase/lite/support/Batcher.java

```diff
@@ -55,7 +65,7 @@ public Batcher(ScheduledExecutorService workExecutor,
         this.processor = processor;
         this.inbox = new ArrayList<T>();
         this.scheduled = false;
-        this.lastProcessedTime = System.currentTimeMillis();
+        this.lastProcessedTime = 0;
```

Comment:
Is this change that makes batcher immediately execute after create new instance of Batcher and then added tasks smaller than capacity size?